app/repositories/FastPassDetailsRepository.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import model.Exceptions.{ CannotUpdateFastPassDetails, FastPassDetailsNotFound }
+import model.FastPassDetails
+import reactivemongo.api.DB
+import reactivemongo.bson.{ BSONDocument, BSONObjectID }
+import uk.gov.hmrc.mongo.ReactiveRepository
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+
+trait FastPassDetailsRepository {
+
+  val FastPassDetailsDocumentKey = "fastpass-details"
+
+  def update(applicationId: String, fastPassDetails: FastPassDetails): Future[Unit]
+
+  def find(applicationId: String): Future[FastPassDetails]
+
+}
+
+class FastPassDetailsMongoRepository(implicit mongo: () => DB) extends
+  ReactiveRepository[FastPassDetails, BSONObjectID]("application", mongo, FastPassDetails.fastPassDetailsFormat,
+    ReactiveMongoFormats.objectIdFormats) with FastPassDetailsRepository {
+
+  override def update(applicationId: String, fastPassDetails: FastPassDetails): Future[Unit] = {
```

Comment:
Are we going to update fastpass details independently from other personal details? It might be good to keep it but as I said in previous comment, maybe this is part of the personal details.